src/objs/basic_types.md

```diff
@@ -0,0 +1,53 @@
+## Basic Object Types
```

Comment:
stick to 'basic object types' or 'primitive object types'

Reply:
yeah i didnt know which one to pick, since i don't know the difference between the two terms really. Which one should i go with?

Reply:
probably primitive